templates/feed.pug

```diff
@@ -1,11 +1,11 @@
-div.container
+div.container.px-0.mb-4
   ul.list-inline.row
     each val in coffeegrams
-      li.list-unstyled.col-sm-4
-        img.img-fluid.rounded(src=`/uploads/${val.image}`)
+      li.list-unstyled.col-sm-4.mb-4
+        img.img-fluid.rounded(src=`/uploads/${val.image}`).mb-2
         br
-        = val.description
+        strong #{val.coffeeType}
         br
-        = val.coffeeType
+        = val.description
         br
-        = val.createdAt
+        small #{val.createdAt}
```

Comment:
If you don't need interpolation, it's conventional to do `small= val.createdAt` instead.